MSAL/src/MSALPublicClientApplication.m

```diff
@@ -442,8 +442,12 @@ - (MSALAccount *)accountForUsername:(NSString *)username
     MSID_LOG_WITH_CTX_PII(MSIDLogLevelInfo, nil, @"Querying MSAL account for username %@", MSID_PII_LOG_EMAIL(username));
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->